keyboards/romac/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```

Comment:
Why are we checking this in?

Reply:
every time I tried to add my files to the repo with git add, it added and edited some of my config files and performed the "LF Normalization" .  I'm super new to this so I have zero clues as to why.